packages/modeling/src/geometries/geom2/toOutlines.js

```diff
@@ -64,31 +74,11 @@ const toOutlines = (geometry) => {
       if (!nextpossiblesides) {
         throw new Error('the given geometry is not closed. verify proper construction')
       }
-      let nextsideindex = -1
```

Comment:
Small request… in the throw above, can we add the vertex? That will provide a little more information for users.

Reply:
added vertex info into the error. much better this way!